spec/features/10_update_reviews_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+feature "Edit Movie" do
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:movie1) { FactoryGirl.create(:movie) }
+  let!(:movie2) { FactoryGirl.create(:movie) }
+  let!(:review5) { FactoryGirl.create(:review, movie_id: movie1.id, user_id: user.id) }
+
+  before do
+    user_sign_in(user)
+  end
+
+  scenario "User clicks link edit review, and is redirected to"\
+    " review edit form " do
+    visit root_path
+    click_link movie1.title
+    # user_review(review5)
+    # binding.pry
+    click_link "Edit Review"
+
+    expect(page).to have_content("Edit Review")
+    click_button "Submit"
+    expect(page).to have_content(review5.body)
+    expect(page).to have_content("Review successfully updated!")
+  end
+
+  scenario "User incorrectly fills out review form and"\
+    " review form is re-renered with error message " do
+    visit root_path
+    click_link movie1.title
+    click_link "Edit Review"
+    expect(page).to have_content("Edit Review")
```

Comment:
It's not best practice to have expect statements in the middle of a test. Tests should have a framework of 1. (optional)set up, 2. action(s), 3. expectations. If you have expectations in the middle of your tests, you're testing multiple things, and you should break them up into separate tests